devops/tasks.py

```diff
@@ -209,3 +224,198 @@ def release(
         logger.info(f"Removing temporary configurations from {rel_path}")
         if rel_path.exists():
             rmtree(rel_path)
+
+
+def get_master_key(env: str) -> None:
+    """
+    Get the master key for SealedSecrets for the given env.
+
+    :param str env: The environment
+    """
+    settings = load_env_settings(env)
+    ensure_context(settings.KUBE_CONTEXT)
+
+    label(logger.info, f"Getting master key for {env}")
+
+    # Based on:
+    # https://github.com/bitnami-labs/sealed-secrets#how-can-i-do-a-backup-of-my-sealedsecrets
+    result = run(
+        [
+            "kubectl",
+            "get",
+            "secret",
+            "-n",
+            "kube-system",
+            "-l",
+            "sealedsecrets.bitnami.com/sealed-secrets-key",
+            "-o",
+            "yaml",
+        ]
+    )
+
+    content = result.stdout.decode(encoding="utf-8")
+    output_file = master_key_path(env=env)
+
+    logger.info(f"Saving master key to {output_file}")
+
+    with output_file.open("w", encoding="utf-8") as f:
```

Comment:
`output_file` is a `Path`, so `output_file.write_text(content, encoding="utf-8")`?

Reply:
Nice, somehow has missed that feature existed. Changes all of them.